taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import UniqueConstraint
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255)
+
+    class Meta:
+        UniqueConstraint(
+            name="license_number",
+            fields=["license_number"],
+        )
+
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(Driver, related_name="car")
```

Comment:
Best practice:
`drivers = models.ManyToManyField(to=settings.AUTH_USER_MODEL, related_name="cars")`